### DISCUSSION TITLE
bStats in extension for minestom

### DISCUSSION THREAD

**Discussion by dev-hydrogen** [Minestom/Minestom#1002] [Help] [answered]
Is it possible to include bstats support in a minestom extension?

**Comment by ZakShearman** [comment, marked as answer]
You'd need to modify the code slightly but yes.

Check out https://bstats.org/getting-started/include-metrics and then you can convert one of their Metrics classes to support Minestom. Keep in mind their requirement of having a bStats config in the extensions folder.

**Reply by dev-hydrogen** [reply]
What about the fact that Minestom is currently not listed as a server implementation on bstats?

**Reply by dev-hydrogen** [reply]
![Discord_rwFUVgru1d](https://user-images.githubusercontent.com/96733109/166114672-45d39c35-704c-48fb-b0ed-9f6a682c4203.png)

Also seems like modifying the metrics class is not allowed

**Reply by ZakShearman** [reply]
Ahh well in that case you would have to ask them
